cli/src/semgrep/output.py

```diff
@@ -382,6 +384,8 @@ def output(
             stats_line = ""
             if print_summary:
                 stats_line = f"\nRan {unit_str(num_rules, 'rule')} on {unit_str(num_targets, 'file')}: {unit_str(num_findings, 'finding')}."
+                if missed_rule_count:
+                    stats_line = f"{stats_line}\nMissed out on {unit_str(missed_rule_count, 'pro rule')}!"
```

Comment:
Maybe not missed? That sounds like an error to me, like a rule failed to run. Maybe unused?

Reply:
Haven't applied the full design treatment, but we should expect "Missed out on" within the copy.

![image](https://github.com/semgrep/semgrep/assets/5779832/f61ef01a-dc75-4adb-8568-e0a5ca0ce8d2)